installlifecycle/lifecycle.go

```
@@ -0,0 +1,514 @@
+			return errors.New("managed entry is not a regular file")
+		}
+		return nil
+	})
+}
+
+func atomicCopy(source, destination string, mode os.FileMode) error {
+	s, err := os.Open(source)
+	if err != nil {
+		return err
+	}
+	defer s.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `s.Close` is not checked

(errcheck)